### PR TITLE
Create LIT6940Incarnation

### PR DESCRIPTION
**Pull request by CarstenHoffmannMarburg**
I created a record for an Amharic commentary in BLorient13223 and EMIP00269 as discussed in https://github.com/BetaMasaheft/Documentation/issues/2488.

### CODE REVIEW

File: new/LIT6940IncarnationAmh.xml [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+        </profileDesc>
+        <revisionDesc>
+            <change who="CH" when="2024-02-23">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <div type="bibliography">
+                <listRelation>
+                    <relation name="lawd:hasAttestation" active="LIT6940IncarnationAmh" passive="BLorient2198"/>
```
Comment (eu-genia) [inline new/LIT6940IncarnationAmh.xml:59]: we usually use the list of witnesses (`listWit `and `witness `under `sourceDesc`) for Works attested in Manuscripts

File: new/LIT6940IncarnationAmh.xml [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+                <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale
+                            Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                                licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <p/>
```
Comment (eu-genia) [inline new/LIT6940IncarnationAmh.xml:28]:
```
<listWit>
<witness corresp="BLorient2198"/>
<witness corresp="EMIP00269"/>
</listWit>
```

**Review by eu-genia — CHANGES_REQUESTED** [review]

File: new/LIT6940IncarnationAmh.xml [hunk 3]
```diff
@@ -25,7 +25,12 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                 </availability>
             </publicationStmt>
             <sourceDesc>
-                <p/>
+                <p>
```
Comment (eu-genia) [inline new/LIT6940IncarnationAmh.xml:28]: you actually do not need the paragraph `p`

**Review by DenisNosnitsin1970 — COMMENTED** [review]
Could you please add translations to the Amharic titles?

**Comment by CarstenHoffmannMarburg** [comment]
> Could you please add translations to the Amharic titles?

I propose: "The unity of God" and "The mysterious trinity and the unity of God"
What translations would you suggest?

Does the translated titly needs a type?

**Comment by DenisNosnitsin1970** [comment]
I would say, rumbling, 
t2 is "The Unity of Divinity and Humanity"
and t3 is "The Unity of the Mystery of Incarnation, Humanity and Divinity".

**Review by DenisNosnitsin1970 — APPROVED** [review]

**Review by eu-genia — APPROVED** [review]

**Pull request merged**